docs/RtspStreamingGuide.md

```diff
@@ -0,0 +1,371 @@
+# USB Camera Device Service RTSP Streaming Guide
+
+## Contents
+
+[System Requirements](#system-requirements)  
+[How It Works](#how-it-works)  
+[Overview](#overview)  
+[Tested Devices](#tested-devices))  
+[Dependencies](#dependencies)  
+[Get the Source Code](#get-the-source-code)
+[Configuration options](#configuration-options)
+[Deploy the Service](#deploy-edgex-and-usb-device-camera-microservice)  
+[Verify the Service](#verify-service-and-device-profiles)   
+[Adding Devices using REST API](#adding-devices-using-rest-api)
+[Start Video Streaming](#start-video-streaming)
+[Shutting Down](#shutting-down)  
+[Troubleshooting](#troubleshooting)) 
+[License](#license)
+
+## System Requirements
+
+- Intel&#8482; Core&#174; processor
+- Ubuntu 20.04.4 LTS
+- USB-compliant Camera
+
+**Time to Complete**
+
+10-20 minutes
```

Comment:
Great to have a time to complete, but not sure what we are completing in this guide. Maybe a section which says what the goal of this document is. Just a simple sentence or two that this document is about streaming from the USB camera  using the RTSP service. That would suffice... Also, how is this different then general-usage.md? Seems like it can all be collapsed into one document... ?

Reply:
@ChristianDarr-personal what do you think of combining this and general-usage or keep them seperate?

Reply:
I think the answer to this hinges on whether or not we want the rtsp guide to be a "walkthrough" or just another step in getting this service running